Readme.md

```diff
@@ -26,6 +30,8 @@ const {
 } = require("path-to-regexp");
 ```
 
+This library is compatible and tested with Node.js 16+ and Bun 1.2+.
```

Comment:
I'd prefer to remove this, it's all just JS so it's compatible with anything that supports ES2015 basically. These just happen to be where tests are run.